src/modules/constants.js

```diff
@@ -0,0 +1,20 @@
+const LOAD_FILMS = 'LOAD_FILMS';
```

Comment:
From my point of view it's better to add an `export` keyword at the begging of line .
```
export const LOAD_FILMS = 'LOAD_FILMS';
...
```
 And you don't need to duplicate constant's names at the end of file 
 ```
 export {
  LOAD_FILMS, CLEAN_FILMS, CHANGE_SEARCH_VALUE, SWITCH_LOADING_FILMS_TO_TRUE, CHANGE_CURRENT_PAGE,
  LOAD_MAIN_FILM, GET_LATEST_FILM_ID, SWITCH_ABOUT_FILM, LOAD_GENRES, SWITCH_ITEM_TO_INFO_MODE,
  SWITCH_ITEM_TO_MAIN_MODE, OPEN_MODAL, CLOSE_MODAL, SWITCH_TAB,
};
```